src/pages/collection/items/GridViewGroupItem.tsx

```diff
@@ -25,20 +25,21 @@ const CountIcon = ({ className, children, show = true }) => (
 const GridViewGroupItem = (item: CollectionGroupType) => {
   const posters = item.Images.Posters;
   const unwatchedCount = item.Sizes.Local.Episodes + item.Sizes.Local.Specials - item.Sizes.Watched.Episodes - item.Sizes.Watched.Specials;
+  const groupCount = item.Sizes.SeriesTypes.Movie + item.Sizes.SeriesTypes.OVA + item.Sizes.SeriesTypes.Other + item.Sizes.SeriesTypes.TV + item.Sizes.SeriesTypes.TVSpecial + item.Sizes.SeriesTypes.Unknown + item.Sizes.SeriesTypes.Web;
 
   return (
     <div key={`group-${item.IDs.ID}`} className="group mr-4 last:mr-0 shrink-0 w-[13.875rem] font-open-sans content-center flex flex-col">
-      <BackgroundImagePlaceholderDiv imageSrc={`/api/v3/Image/${posters[0].Source}/Poster/${posters[0].ID}`} className="h-[19.875rem] rounded drop-shadow-[0_4px_4px_rgba(0,0,0,0.25)] border border-black my-2 relative" hidePlaceholderOnHover={true}>
+      <BackgroundImagePlaceholderDiv imageSrc={`/api/v3/Image/${posters[0].Source}/Poster/${posters[0].ID}`} className="h-[19.875rem] rounded drop-shadow-[0_4px_4px_rgba(0,0,0,0.25)] border border-black my-2 mb-3 relative" hidePlaceholderOnHover={true}>
```

Comment:
if you are specifying `mb-3`, also change `my-2` to `mt-2` instead

Reply:
Whoops, meant to remove ``my-2`` altogether.